homework-submission/myapp/src/stores/todoStore.js

```diff
@@ -0,0 +1,49 @@
+import { observable, action } from "mobx";
+import Data from "./../data/data";
+
+class todoStore {
+  @observable Data = Data;
+
+  @action handleCheckBox = id => {
+    this.Data = this.Data.map(item => {
```

Comment:
I don't see any use of `.map` here more than the iteration part, while we are looping through all the elements and mutating by there value, we are mutating them directly and not by reference.

with that using `.forEach` is a bit more reasonable, same as the next method.